### DISCUSSION TITLE
Explain me Curl use Cookie jar file in Script

### DISCUSSION THREAD

**Discussion by MohdRashid01** [curl/curl#7816] [Q&A] [answered]

**Comment by jzakrzewski** [comment]
`file.sh` is a script written by SOMEONE. It just happens to use curl to achieve something - I don't understand what, really, but the goal seems to be to obtain XSRF_TOKEN.
Th cookie_jar is just a file where curl stores the cookies that were used in the above sequence of commands. Typically there's nothing interesting to the user there.

**Comment by bagder** [comment, marked as answer]
[The cookie jar file format](https://everything.curl.dev/libcurl-http/cookies#cookie-file-format)
